docs/docs/api/nativeMethods/measure.md

```diff
@@ -4,12 +4,29 @@ title: measure
 sidebar_label: measure
 ---
 
-> Due to time constraints we weren't able to finish this page.
+Determines the location on screen, width, and height of the given view. Note that these measurements are not available until after the rendering has been completed in native. If you need the measurements as soon as possible, consider using `onLayout` instead.
```

Comment:
Could we explain what is `onLayout` or add a link to documentation?

Reply:
fixed 787d616